pom.xml

```diff
@@ -23,7 +24,7 @@
   <repositories>
     <repository>
       <id>mycbr-repo</id>
-      <url>file://${basedir}/lib</url>
+      <url>file://${basedir}${file.separator}lib</url>
```

Comment:
Does the correct file separator comes from maven? Is it tested on other OS than windows?

Reply:
Maven exposes all properties from `java.lang.System` which includes file.seperator.
I've successfully compiled the project under Windows 10 as well as Linux (Manjaro)